packages/astro/src/integration/create-integration.ts

```diff
@@ -111,7 +111,7 @@ function createIntegration<Params extends HotloadAstroClerkIntegrationParams>()
             ${command === 'dev' ? `console.log("${packageName}","Initialize Clerk: page")` : ''}
             import { runInjectionScript, swapDocument } from "${buildImportPath}";
 
-            // Taken from https://github.com/withastro/astro/blob/e10b03e88c22592fbb42d7245b65c4f486ab736d/packages/astro/src/transitions/router.ts#L39.
+            // Taken from https://github.com/withastro/astro/blob/f13417bfbf73130c224752379e2da33084f89554/packages/astro/src/transitions/router.ts#L39.
```

Comment:
why do we need to update the url ?

Reply:
I think the file has been updated and I think it would make sense to get updated source by the time we release new major